site-capstone/Backend/models/notification.js

```diff
@@ -0,0 +1,89 @@
+const db = require("../db");
+const twilio = require('twilio');
+const {TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN, TWILIO_PHONE_NUMBER} = require("../config")
+const { BadRequestError, NotFoundError } = require("../utils/errors");
+const schedule = require('node-schedule');
+
+// twilio account API specfic information (keep hidden)
+const client = twilio(TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN);
+
+// sendText function creates a text using twilio function
+function sendText(message, phone_number){
+    client.messages 
+        .create({ 
+            // what appears in the text when sent
+            body: `${message}`,       
+            // twilios number
+            from: TWILIO_PHONE_NUMBER,
+            // users inputted number need country code to work
+            to: `${phone_number}`
+            }) 
+            // console logs specfic message number to see if sent
+        .then(message => console.log(message.sid)) 
```

Comment:
Need to delete or comment out, consider 'console.error' for this.

Reply:
Using twilio api's since it sends messages we cant tell if it actually if it gets sent without receiving it on the phone the console.log just gives us a message id to tell if  it gets sent. Should we still go about removing it?

Reply:
Just use 'console.error' and we should be good to go.

Reply:
Nvm, I see what you mean

Reply:
What about revealing a 'Sent Successfully' message? You can have it as an alert and then implement it as a CSS class to reveal it.

Reply:
we just removed it. It was just to see if it failed at our server end or twilio end. The way its set up acts as an error message tells exactly what part is wrong within the create message function